modules/bitgo/example/js/btc/withdraw-lightning-balance.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Withdraw from wallet's lightning balance to on-chain funds.
+ *
+ * Copyright 2022 BitGo, Inc.  All Rights Reserved.
+ */
+const BitGoJS = require('bitgo');
+
+// change this to env: 'production' when you are ready for production
+const env = 'test';
+// change coin to 'btc' when working with production
+const coin = 'tbtc';
+
+const bitgo = new BitGoJS.BitGo({ env });
+
+// set your access token here
+const accessToken = '';
+
+// set your wallet from the YYYYY parameter here in the URL on app.bitgo-test.com
+// https://test.bitgo.com/enterprise/XXXXXXXXX/coin/tbtc/YYYYY/transactions
+const walletId = '';
+
+async function withdrawLightningBalance() {
+  bitgo.authenticateWithAccessToken({ accessToken });
+  const wallet = await bitgo.coin(coin).wallets().get({ id: walletId });
+  const lightning = wallet.lightning();
+
+  const initialLightningBalance = await lightning.getBalance();
+  console.log('Initial Lightning Balance:', initialLightningBalance.balance);
+  const initialBalance = wallet.balance();
+  console.log('Initial Balance:', initialBalance);
+
+  const fixtureWithdraw = await lightning.withdraw({ value: 10000 });
```

Comment:
Why is this called `fixtureWithdraw`? I think it could just be `withdrawResponse`.